README.md

```diff
@@ -13,9 +13,9 @@ A manager for BetterDiscord on Linux.
 
 ### Manual
 
-Requires `git` which you can install from your distro's [package manager][git-packages].
+Requires `curl` which you can install from your distro's [package manager][curl-packages].
 
-[git-packages]:  https://git-scm.com/download/linux/
+[curl-packages]: https://curl.se/download.html#Linux
```

Comment:
We previously depended on curl. It's probably good that we're documenting it now, but just noting. It's a lot more common than Git in base installs, but I've also seen it not installed by default, so :woman_shrugging:.